qutip/tests/test_brtools.py

```diff
@@ -121,17 +121,17 @@ def test_vector_roundtrip():
                                    atol=1e-12)
 
 
-def test_diag_liou_mult():
+@pytest.mark.parametrize('dimension', list(range(2, 100)))
+def test_diag_liou_mult(dimension):
     "BR Tools : Diagonal Liouvillian mult"
-    for dimension in range(2, 100):
-        H = qutip.rand_dm(dimension, 0.5)
-        evals, evecs = H.eigenstates()
-        L = qutip.liouvillian(H.transform(evecs))
-        coefficients = np.ones((dimension*dimension,), dtype=np.complex128)
-        calculated = np.zeros_like(coefficients)
-        target = L.data.dot(coefficients)
-        _test_diag_liou_mult(evals, coefficients, calculated, dimension)
-        np.testing.assert_allclose(target, calculated, atol=1e-12)
+    H = qutip.rand_dm(dimension, 0.5)
+    evals, evecs = H.eigenstates()
+    L = qutip.liouvillian(H.transform(evecs))
+    coefficients = np.ones((dimension*dimension,), dtype=np.complex128)
+    calculated = np.zeros_like(coefficients)
+    target = L.data.dot(coefficients)
+    _test_diag_liou_mult(evals, coefficients, calculated, dimension)
+    np.testing.assert_allclose(target, calculated, atol=1e-11, rtol=1e-6)
```

Comment:
Just noting that the changed tolerances here are the core of the fix (in case other reviewers want to comment on them).

Reply:
Oh yeah - it's clearer if you look at the two commits separately, rather than using the "squashed" patch GitHub shows by default.